android/java/org/mlperf/inference/MLPerfTasks.java

```diff
@@ -32,6 +33,9 @@
 public final class MLPerfTasks {
   private static final String TAG = "MLPerfTasks";
   private static final String ZIP = ".zip";
+  private static final String LOCAL_FOLDER = "/sdcard/mlperf_results/mlperf/";
+  private static final String LOCAL_CACHE_FOLDER = LOCAL_FOLDER + "cache/";
```

Comment:
LOCAL_CACHE_FOLDER will be `/sdcard/mlperf_results/mlperf/cache`. The path is a bit unusual. Should LOCAL_CACHE_FOLDER better be `/sdcard/mlperf_cache`?